pkix/csr.go

```diff
@@ -52,14 +52,16 @@ var (
 )
 
 // ParseAndValidateIPs parses a comma-delimited list of IP addresses into an array of IP addresses
-func ParseAndValidateIPs(ipList string) (res []net.IP) {
+func ParseAndValidateIPs(ipList string) (res []net.IP, err error) {
 	ips := strings.Split(ipList, ",")
 	for _, ip := range ips {
-		parsedIP := net.ParseIP(ip)
-		if parsedIP == nil {
-			return nil
+		if len(ip) > 0 {
```

Comment:
what is the purpose of the len(ip) check (i.e. when is it 0)? Maybe add a comment in the code?